File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/C/Exercicio 7.c

/*
7 - Elabore um programa em C para calcular e escrever a seguinte soma:
    37 * 38       36 * 37      35 * 36            1 * 2
S = ---------- + --------- + ---------- + ..... + ------
        1           2           3                   37      
*/

#include<stdio.h>

int main ()
{
    float soma=0;
    int den, num1=37, num2=38;

    for (den=1; den<=37; den++){
        soma += ((num1*num2)/(float)den);
        num1--;
        num2--;
    }
    printf ("\n\n A soma eh: %.2f\n\n",soma);
    return 0;
}

File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/Python/exercicio7.py
"""7 - Elabore um programa para calcular e escrever a seguinte soma:
    37 * 38       36 * 37     35 * 36             1 * 2
S = ---------- + --------- + ---------- + ..... + ------
        1           2           3                   37
"""

soma=0
numerador1=37
numerador2=38
denominador=1

for denominador in range(1,37+1):
  soma += (float) ((numerador1*numerador2)/denominador)
  numerador1 -= 1
  numerador2 -= 1

print('\nResultador da somatória: '+str(soma))
